Device/ARM/ARMv8MML/Source/system_ARMv8MML.c

```diff
@@ -79,7 +79,7 @@ void SystemInit (void)
 {
 
 #if defined (__VTOR_PRESENT) && (__VTOR_PRESENT == 1U)
-  SCB->VTOR = (uint32_t) &__Vectors;
+  SCB->VTOR = (uint32_t)(&__VECTOR_TABLE);
```

Comment:
Good catch!

Reply:
As Gabriel suggests we might think about moving the brackets to make the macro evaluation more safe: `(uint32_t)&(__VECTOR_TABLE)`.

Reply:
I noticed that only a few of the system_ARM*.c files had parenthesis around __VECTOR_TABLE, should they have or should they be removed?

Reply:
The suggestion is to have the parentheses surrounding the macro directly to assure proper macro expansion. In fact I am not sure under which circumstances this might fail. Perhaps @GorgonMeducer can give us an example?